lib/pf/action.pm

```diff
@@ -353,15 +355,15 @@ sub action_reevaluate_access {
 sub action_autoregister {
     my ($mac, $vid) = @_;
     my $logger = get_logger();
-
+    my $unregdate = access_duration($pf::violation_config::Violation_Config{$vid}{access_duration});
```

Comment:
What if its not defined ? Is it still making it registered for life ?

Reply:
The question is do we allow this behavior or not.